reactor-netty-http/src/main/java/reactor/netty/http/server/MicrometerHttpServerMetricsRecorder.java

```diff
@@ -37,9 +43,24 @@
 final class MicrometerHttpServerMetricsRecorder extends MicrometerHttpMetricsRecorder implements HttpServerMetricsRecorder {
 
 	final static MicrometerHttpServerMetricsRecorder INSTANCE = new MicrometerHttpServerMetricsRecorder();
+	private final static String HTTP = "http";
+
+	protected final Counter.Builder activeConnectionsBuilder;
+	protected final ConcurrentMap<MeterKey, Counter> activeConnectionsCache = new ConcurrentHashMap<>();
+
+	protected final Counter.Builder connectionsBuilder;
+	protected final ConcurrentMap<MeterKey, Counter> connectionsCache = new ConcurrentHashMap<>();
 
 	private MicrometerHttpServerMetricsRecorder() {
-		super(HTTP_SERVER_PREFIX, "http");
+		super(HTTP_SERVER_PREFIX, HTTP);
+
+		this.activeConnectionsBuilder =
```

Comment:
same as in TCP, use a `Gauge` over `AtomicInteger` (unless we think integer max value is not enough in practice, in which case `AtomicLong`)